microshift-okd/src/build.sh

```diff
@@ -0,0 +1,41 @@
+#/bin/bash
+
+
+# replace microshift assets to Upsteam from OKD
+okd_url=quay.io/okd/scos-release
+okd_releaseTag=4.17.0-0.okd-scos-2024-08-21-100712
+
+oc adm release info ${okd_url}:${okd_releaseTag} >all_images
+
+for op in $(cat assets/release/release-x86_64.json  | jq -e -r  '.images | keys []') 
+do
+    image=$(oc adm release info --image-for=${op} ${okd_url}:${okd_releaseTag} || true) 
+    if [ -n "${image}" ] ; then
+        echo "${op} ${image}"
+        jq --arg a "${op}" --arg b "${image}"  '.images[$a] = $b' assets/release/release-x86_64.json >/tmp/release-x86_64.json.tmp
+
+
+        # delete openssl image from assets - just to verify if we still need it,since it doesnt referenced anywhere
+        jq '. | del (.images["openssl"])'  assets/release/release-x86_64.json >/tmp/release-x86_64.json.tmp2
```

Comment:
We are removing the `openssl` dependency in https://github.com/openshift/microshift/pull/3874.
Let's drop this code once the PR is merged.

Reply:
since we are based on 4.17
this is the relevant change  https://github.com/openshift/microshift/pull/3884